content/guides/tasks/update.md

```diff
@@ -0,0 +1,15 @@
+---
+rank: 3
+related_endpoints:
+  - put_tasks_id
+related_guides:
+required_guides: []
+alias_paths: []
+---
+
+# Make changes to an existing task
+
+To update a task in Box you will need to call the
+[`PUT /tasks/:task_id`](e://put_tasks_id) API.
+
+<Samples id='put_tasks_id' />
```

Comment:
I think we should expand this to cover some of the params you can use here to change the message, completion rule, and more.